backend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9-slim
+
+COPY . /opt/aimaas
```

Comment:
Personally, I don't like installing Python packages this way, but on the other hand I did not want to refactor the project into a nice package structure when there are still pending changes. IMHO, this is the lesser of two evils.